reconcile/utils/vault.py

```diff
@@ -115,13 +118,25 @@ def read_all(self, secret):
         if kv_version == 2:
             data = self._read_all_v2(secret_path, secret_version)
         else:
-            data = self._read_all_v1(secret_path)
+            secret_data = self._read_all_v1(secret_path)
+            data = secret_data, None
```

Comment:
I think we talked about this or a I put it in a comment, but I'd honestly prefer that `data` is still a dict and this function explicitly returned a tuple in the form of

```
return data, version
```

So that it is even clearer how different it is from `read_all`.

Please also fix the docstring, which is no longer exact.

Reply:
Fixed both to match the current usecase